.travis.yml

```diff
@@ -8,15 +8,36 @@ sudo: false
 addons:
   apt:
     packages:
+    - bc
+    - doxygen
+    - expect
+    - gettext
+    - libncurses5-dev
     - tree
-    - fish
+
+env:
+  global:
+    - PATH="$HOME/fish/bin:$PATH"
+    - OMF_REPO_URI="https://github.com/$TRAVIS_REPO_SLUG"
+    - OMF_REPO_BRANCH="$TRAVIS_BRANCH"
+
+before_install:
+  - tests/install_fish.sh
+  - fish -c 'ln -sf $TRAVIS_BUILD_DIR/tests/fish_user_function $fish_function_path[1]'
```

Comment:
This is to point the directory `~/.config/fish/fish_user_function` on the one present in the tests dir.
It allow us to override some function which may not work in the travis build. (like refresh).